### ISSUE TITLE
Add pixel 7 wallpapers

### ISSUE THREAD

**Issue by Nitroandaccounts** [patzly/doodle-android#82]
I know it takes a while to implement, but maybe consider adding this.

**Comment by patzly** [comment]
Yeah they are wallpapers, but as I said this app is only about doodle-styled shape-based wallpapers.

**Comment by Nitroandaccounts** [comment]
Ah, I see. It is pretty much that you do what other developers won’t do, which is create high quality scaled wallpapers. The rest you can find online. Do you have a donation page? PayPal? Patreon?

**Comment by patzly** [comment]
Yes you're right, that's why I cannot do that with photos: the photos are not scalable (and not good for my svg format) like the svg shapes of the other wallpapers :)
You can find me on PayPal: [@pzedler](https://www.paypal.me/pzedler). Thank you so much!

**Comment by Nitroandaccounts** [comment]
Thank you. I will donate as soon as I am on my phone. Additionally, I noticed you have android 13 optimization in your todo. Does this app not work with android 13?

**Comment by Dyterul** [comment]
I am using the wallpapers on my Pixel 7 running Android 13 and they work just fine. There are of course some minor bugs like zoom out on lockscreen that jerks slightly when locking the device. But in general, the wallpapers work very well and look fantastic.

**Comment by Nitroandaccounts** [comment]
Ah ok

**Comment by Nitroandaccounts** [comment]
> Yes you're right, that's why I cannot do that with photos: the photos are not scalable (and not good for my svg format) like the svg shapes of the other wallpapers :)
> You can find me on PayPal: [@pzedler](https://www.paypal.me/pzedler). Thank you so much!

Any chance you have a crypto wallet, PayPal fees are exuberant

**Comment by Nitroandaccounts** [comment]
Or a kofi page

**Comment by patzly** [comment]
Can't you select "friends or family"? Other users who have donated had selected that I think xD
No, I have no crypto wallet :)

**Comment by patzly** [comment]
@Dyterul Thank you so much for reading the issues! In fact, I have pretty much optimized all of the app for Android 13 I think, currently I have no Android 13-specific todos anymore. The next update is already finished and will hopefully make the tilt and zoom effect much smoother!

**Comment by Nitroandaccounts** [comment]
> Can't you select "friends or family"? Other users who have donated had selected that I think xD
> No, I have no crypto wallet :)

It gives a 1.50 US dollar tax. Maybe bc I’m not in your country?

**Comment by patzly** [comment]
That's the reason I think :/
